Doc/library/codecs.rst

```diff
@@ -1106,8 +1106,7 @@ particular, the following variants typically exist:
 +-----------------+--------------------------------+--------------------------------+
 | cp1258          | windows-1258                   | Vietnamese                     |
 +-----------------+--------------------------------+--------------------------------+
-| cp65001         |                                | Windows only: Windows UTF-8    |
-|                 |                                | (``CP_UTF8``)                  |
+| cp65001         |                                | Alias to ``utf_8`` encoding    |
```

Comment:
Add the versionchanged directive.

I think it is better to remove this row add add cp65001 to the list of utf-8 aliases.

Reply:
I created PR #13240 for your doc change proposal.